### DISCUSSION TITLE
Why tests, which have been passed, in Allure report are displayed in the Unknown status?

### DISCUSSION THREAD

**Discussion by InnaOrtman** [allure-framework/allure2#1998] [Questions & Support]
Tests - that have passed (in spec reporter, as well) a few days ago, for no particular reason, because I have not updated either the tests themselves or Allure Reporter, - began to be displayed in the Unknown status!
<img width="1494" alt="Снимок экрана 2023-05-25 в 17 09 37" src="https://github.com/allure-framework/allure2/assets/84459455/0d77cc8e-38d9-4f9c-bbf1-b3af52dea849">

**Comment by vinceyapTZ** [comment]
In my case Allure was showing Unknown for a passed test when there was a warning in the console log: `WARN webdriver: Request encountered a stale element - terminating request`
